experiments/voxel/enviroment.Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN cd /; \
     git clone https://github.com/paradise-fi/RoFI.git --depth 1 --branch master; \
     cd RoFI; \
     source ./setup.sh Release; \
+    # Uncomment if you want to enable debug assertions in Rust
+    # export RUSTFLAGS="-C debug-assertions -C overflow-checks"; \
```

Comment:
Nemohli bychom na toto přidat env proměnnou, kterou si vložíš do `.rofi.pre`?

Reply:
Popravdě jsem přemýšlel, že od tohohle je RelWithDeb - a jestli to půjde, tak to zakomponuju do toho

Reply:
Fair point. Dát to do RelWithDeb se mi líbí nejvíce. Změníš to je ještě prosím?